src/seedcase_sprout/core/sprout_checks/check_data_header.py

```diff
@@ -0,0 +1,32 @@
+import polars as pl
+
+from seedcase_sprout.core.get_nested_attr import get_nested_attr
+from seedcase_sprout.core.properties import ResourceProperties
+
+
+def check_data_header(
+    data_frame: pl.DataFrame, resource_properties: ResourceProperties
```

Comment:
```suggestion
    data: pl.DataFrame, resource_properties: ResourceProperties
```

To match what the argument is called in other functions.